fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -2888,9 +2888,10 @@ public void unsetFields() {
         // Has to use synchronized to ensure there are not concurrent update threads. Or the done
         // state maybe update wrong and will lose data. see https://github.com/apache/doris/pull/29802/files.
         public synchronized boolean updatePipelineStatus(TReportExecStatusParams params) {
-            // The fragment or instance is not finished, not need update
+            // The fragment or instance is not finished, still need handle the report to update the load progress.
+            // Otherwise the load progress (ScannedRows/LoadBytes) won't be updated before any instance is done.
             if (!params.done) {
-                return false;
+                return true;
```

Comment:
This may bring some problems here, refer to the comments above.

Reply:
reverted